src/core/public/http/http_service.test.ts

```diff
@@ -459,7 +459,7 @@ describe('interception', () => {
   });
 
   describe('request availability during interception', () => {
-    it('should not be available to responseError when request throws', async () => {
+    it('should be available to responseError when request throws', async () => {
```

Comment:
I missed this behavior before.... Why are we calling an interceptor's `responseError` when there's an error throw during `request` interception?